src/test/UserTest/UserControllerIntegrationTest.java

```diff
@@ -50,6 +50,42 @@ public void loginUserWithNoUsernameTest() {
     assertThat(actualResponseJSON.getString("status")).isEqualTo("AUTH_FAILURE");
   }
 
+  @Test
+  public void testUserEncryption() {
+    TestUtils.login("adminBSM", "adminBSM");
+    JSONObject body = new JSONObject();
+    body.put("username", "adminBSM");
+
+    HttpResponse<String> actualResponse =
+        Unirest.get(TestUtils.getServerUrl() + "/get-user-info").asString();
+    JSONObject actualResponseJson = TestUtils.responseStringToJSON(actualResponse.getBody());
+    assertThat(actualResponseJson.get("city").toString()).isEqualTo("Philadelphia");
+    assertThat(actualResponseJson.get("firstName").toString()).isEqualTo("Mike");
+    assertThat(actualResponseJson.get("lastName").toString()).isEqualTo("Dahl");
+    assertThat(actualResponseJson.get("zipcode").toString()).isEqualTo("19104");
+    assertThat(actualResponseJson.get("phone").toString()).isEqualTo("1234567890");
+    assertThat(actualResponseJson.get("address").toString()).isEqualTo("311 Broad Street");
+    assertThat(actualResponseJson.get("birthDate").toString()).isEqualTo("06-16-1960");
+    assertThat(actualResponseJson.get("email").toString())
+        .isEqualTo("mikedahl@broadstreetministry.org");
+  }
+
+  @Test
+  public void testGetMembersEncryption() {
+    TestUtils.login("adminBSM", "adminBSM");
+    JSONObject body = new JSONObject();
+    body.put("name", "Mike Dahl");
+    body.put("listType", "clients");
+    body.put("currentPage", "1");
+    body.put("itemsPerPage", "10");
+
+    HttpResponse<String> actualResponse =
+        Unirest.post(TestUtils.getServerUrl() + "/get-organization-members")
+            .body(body.toString())
+            .asString();
+    JSONObject actualResponseJson = TestUtils.responseStringToJSON(actualResponse.getBody());
```

Comment:
how does this test encryption at all?

Reply:
It doesn't. This is an old test, that will now fail

Reply:
it is a sad thing